scripts/run-ci-e2e-tests.js

```diff
@@ -70,6 +72,25 @@ try {
 
   const REACT_NATIVE_PACKAGE = path.join(ROOT, 'react-native-*.tgz');
 
+  describe('Verdaccio');
+  const verdaccioProcess = child_process.spawn('npx', [
+    'verdaccio',
```

Comment:
I didn't see this added to dependencies, so I think we want to lock the version somehow, you can do it like this:
```suggestion
    'verdaccio@5.15.3',
```